options/locale/locale_en-US.ini

```diff
@@ -2840,6 +2840,14 @@ settings.location = Location
 settings.permission = Permissions
 settings.repoadminchangeteam = Repository admin can add and remove access for teams
 settings.visibility = Visibility
+settings.change_visibility = Change Visibility
+settings.invalid_visibility = The new visibility is not valid.
+settings.change_visibility_notices_1 = This operation <strong>CANNOT</strong> be undone.
+settings.change_visibility_notices_2 = Non-members will lose access to the organization’s repositories if visibility is changed to private.
+settings.change_visibility_no_change = You did not make any changes to visibility.
+settings.change_visibility_failed = Failed to change the visibility of %s due to an internal error.
```

Comment:
In the future, we need to introduce various "failed ... due to an internal error" messages for every route handler?

Reply:
Why such message is needed?

"Internal error" should never (seldom) happen to users, even they see the message, the message helps nothing and they can do nothing.

Reply:
ecc5b7280041d9809645a192aa99f0afe59af488